tests/unit/lms/services/h_api_test.py

```diff
@@ -16,42 +15,39 @@
 from lms.services.h_api import HAPI
 from lms.values import HUser
 
-BASE_URL = "http://example.com/base/"
-AUTHORITY = "test-authority"
-
 
 class TestHAPI:
 
     # We're accessing h_api._api_request a lot in this test class, so disable
     # protected-access messages.
     # pylint: disable=protected-access
 
-    def test_get_user_works(self, h_api):
-        h_api._api_request.return_value.json.return_value = {
+    def test_get_user_works(self, h_api, _api_request):
```

Comment:
The new `_api_request` fixture is a mocked/patched version of `HAPI._api_request()`, and is used by all the tests that want a mocked version of that method to assert against. Tests that _don't_ want `_api_request()` to be mocked just don't use this fixture